webpack.config.js

```diff
@@ -4,6 +4,7 @@ var NODE_ENV = process.env.NODE_ENV || 'development';
 const { ProvidePlugin } = require( 'webpack' );
 const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
 const WordPressExternalDependenciesPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const webpack = require( 'webpack' );
```

Comment:
I am not certain will it work, but can you maybe require `DefinePlugin` from webpack instead of the whole webpack?

Reply:
Fixed, thanks for catching it.